src/core/system_parameters/beidou_cnav2_almanac.h

```diff
@@ -0,0 +1,113 @@
+/*!
+ * \file beidou_cnav2_almanac.h
+ * \brief  Interface of a beidou cnav2 ALMANAC storage
+ * \note Code added as part of GSoC 2017 program
+ * \author Dong Kyeong Lee, 2018. dole7890(at)colorado.edu
+ * \see <a href="http://m.beidou.gov.cn/xt/gfxz/201712/P020171226742357364174.pdf">beidou ICD</a>
+ *
+ * -------------------------------------------------------------------------
+ *
+ * Copyright (C) 2010-2015  (see AUTHORS file for a list of contributors)
+ *
+ * GNSS-SDR is a software defined Global Navigation
+ *          Satellite Systems receiver
+ *
+ * This file is part of GNSS-SDR.
+ *
+ * GNSS-SDR is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * GNSS-SDR is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GNSS-SDR. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * -------------------------------------------------------------------------
+ */
+
+
+#ifndef GNSS_SDR_beidou_ALMANAC_H_
```

Comment:
`#define` safeguards should be in capital letters. Should use
`#ifndef GNSS_SDR_BEIDOU_ALMANAC_H_`